app/controllers/concerns/github_repo_status.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module GitHubRepoStatus
```

Comment:
+1 for using a concern here. Assignment and group-assignment are super similar, we should be using these more often.